test/test_contract.js

```diff
@@ -21,6 +22,17 @@ describe('Chess contract', function() {
   const player2 = web3.eth.accounts[1];
   const player3 = web3.eth.accounts[2];
 
+  // Remove this for CI/deploy, otherwise the test never finishes
```

Comment:
Maybe, but it's not that easy when a test fails. In any case, we don't really want to litter the test with debug outputs. It's only useful while writing the test.